router/utils/utils.go

```diff
@@ -41,13 +41,22 @@ func loadConfig() {
 	config.RegisterDurationConfigVariable(time.Duration(720), &JobRetention, true, time.Hour, "Router.jobRetention")
 }
 
+func getRetentionTimeForDestination(destID string) time.Duration {
+	destJobRetentionFound := config.IsSet("Router." + destID + ".jobRetention")
```

Comment:
make `destID` value safe for usage on yaml or env variables.

Reply:
We already do this in a bunch of places. If needed, clean up all at once later?

Reply:
For converting a config property to an environment variable what we do now is:

1. Convert from camel case to snake case
2. Replace `.` with `_`
3. prepend `RSERVER_`


Here is an excerpt from the documentation of a [configuration management spec](https://download.eclipse.org/microprofile/microprofile-config-2.0-M1/microprofile-config-spec.html#default_configsources.env.mapping) used in java, which I find insightful. We may follow the same (or similar) approach:

> Some operating systems allow only alphabetic characters or an underscore, _, in environment variables. Other characters such as ., /, etc may be disallowed. In order to set a value for a config property that has a name containing such disallowed characters from an environment variable, the following rules are used.
> 
> The ConfigSource for the environment variables searches three environment variables for a given property name (e.g. com.ACME.size):
>
> Exact match (i.e. com.ACME.size)
>
> Replace each character that is neither alphanumeric nor _ with _ (i.e. com_ACME_size)
> 
> Replace each character that is neither alphanumeric nor _ with _; then convert the name to upper case (i.e. COM_ACME_SIZE)

>

Reply:
Good point. We will clean up later.